src/test/run-make/const_fn_mir/dump.mir

```diff
@@ -40,3 +24,19 @@ fn foo() -> i32 {
         return;                          // scope 0 at main.rs:6:2: 6:2
     }
 }
+
+fn main() -> () {
```

Comment:
Not sure if I was supposed to bless this test, or if it represents an error...

Reply:
This is fine!